src/realm.h

```diff
@@ -3894,6 +3898,12 @@ RLM_API realm_sync_session_connection_state_notification_token_t* realm_sync_ses
     realm_sync_session_t*, realm_sync_progress_func_t, realm_sync_progress_direction_e, bool is_streaming,
     realm_userdata_t userdata, realm_free_userdata_func_t userdata_free) RLM_API_NOEXCEPT;
 
+
+// register callback for tracking user status.
```

Comment:
// @return a notification token object. Dispose it to stop receiving notifications.